### ISSUE TITLE
[TOOL IMPROVEMENT] QR to generate Wi-Fi access codes and others.

### ISSUE THREAD

**Issue by voronind-com** [CorentinTh/it-tools#633]
**Describe the solution you'd like**
I'd like an existing QR code generator to generate additional features like Wi-Fi connection, similar to how Android phones and NFC tools do that. You enter SSID and password and it generates QR code that anyone can scan and connect to that AP. IIRC there are more such standards, but I can't name more at the moment.

**Additional context**
-

**Comment by sharevb** [comment]
Hi @voronind-com, sounds like https://it-tools.tech/wifi-qrcode-generator

**Comment by sharevb** [comment]
Hi @voronind-com , implemented in my fork : https://sharevb-it-tools.vercel.app/wifi-qrcode-generator (may need Ctrl+F5)

And if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
and I integrated your fix

**Comment by voronind-com** [comment]
Yeah, thanks for all this! It works great. Sorry for the late close.
